src/Sulu/Component/Content/Metadata/Parser/PropertiesXmlParser.php

```diff
@@ -154,7 +153,7 @@ private function loadBlock(\DOMXPath $xpath, \DOMNode $node, &$tags)
                 'default-type',
                 'minOccurs',
                 'maxOccurs',
-                'colspan',
+                'colSpan',
```

Comment:
Shouldn't that be left to `colspan`? Isn't reading the value from the XML, where we've decided to keep it this way?

Reply:
I think you are right here. Should be ok now.